tests/foreman/ui/test_organization.py

```diff
@@ -30,24 +30,23 @@
 
 
 @pytest.fixture(scope='module')
-def module_repos_col(module_org, module_lce, module_target_sat, request):
-    module_target_sat.upload_manifest(org_id=module_org.id)
+def module_repos_col(module_entitlement_manifest_org, module_lce, module_target_sat, request):
```

Comment:
Since you are touching this line, please bring `request` as the first parameter as a good practice of coding :)